### DISCUSSION TITLE
List of what the "status bar" icons mean

### DISCUSSION THREAD

**Discussion by ryester19** [AnySoftKeyboard/AnySoftKeyboard#2700] [Q&A]
Is there a list of what the status bar icons mean on the bottom of the keyboard? I figured out that the purple squiggly line is for gesture typing being enabled, but I can't figure out what the orange flame icon is for

**Comment by menny** [comment]
UX is not the best part in AnySoftKeyboard, evidently.
You are right about the gesture typing.
The flame means you're running a beta/alpha version.
The eye with a cross means Incognito mode is on.
A robot/android icon means you are using a build from a local build.

**Reply by menny** [reply]
Can you suggest a way to educate users about those?

**Reply by Snuffleboot** [reply]
How about having a legend/ card in the settings "Home" tab, with a short explanation ("Incognito mode on" for example) beneath the icon? Not the best idea but perhaps worth considering until you find a better solution?
